src/rosinstall_generator/generator.py

```diff
@@ -249,11 +251,14 @@ def generate_rosinstall(distro_name, names,
         logger.debug("The following wet packages found in '--from-path' will be considered: %s" % ', '.join(sorted(include_names_from_path)))
         names.update(include_names_from_path)
 
+    # Allow special keywords in repos (like ALL)
+    repo_names, keywords = _split_special_keywords(repo_names or [])
```

Comment:
If `RPP` would be in the passed argument should that be flagged as an error here? Currently it is silently ignored.

Reply:
Raises `RuntimeError` if any keyword besides `ALL` is present in 4bf324ad4fa40077f3be1a43cc963662a1db2133